dandiapi/api/views/asset.py

```diff
@@ -82,6 +83,11 @@ class AssetViewSet(DetailSerializerMixin, GenericViewSet):
     filter_backends = [filters.DjangoFilterBackend]
     filterset_class = AssetFilter
 
+    def get_throttles(self) -> list[BaseThrottle]:
+        if self.action == 'list':
+            return [*self.throttle_classes, AnonRateThrottle()]
+        return super().get_throttles()
```

Comment:
To prevent some weird bug in the future, since as is, any existing throttles in `self.throttle_classes` will cause the error you just fixed:

```suggestion
        if self.action == 'list':
        	throttles = [*self.throttle_classes, AnonRateThrottle]
            return [throttle() for throttle in throttles]
        return super().get_throttles()
```